page-sitemap.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Template Name: SiteMap
+ *
+ * The template for displaying a sitemap page.
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package nightingale-wp
+ */
+
+get_header(); ?>
+<div class="o-layout  o-layout--wide">
+
+	<div id="primary" class="o-layout__item u-8/12@lg">
+
+		<main id="main" class="site-main" role="main">
+
+			<h2 id="pages">Pages</h2>
+						<ul>
```

Comment:
Indentation seems a bit off? Can we tidy this up please?

Reply:
Done.